gcache/src/gcache_rb_store.cpp

```diff
@@ -1035,10 +1035,6 @@ namespace gcache
             else assert(size_trail_ >= sizeof(BufferHeader));
 
             estimate_space();
-            /* On graceful shutdown all the active buffers are released
```

Comment:
This isn't a conflict resolution? Please explain/mention in the commit message.

Reply:
Fixed